torch/ao/quantization/_quantize_pt2e.py

```diff
@@ -0,0 +1,205 @@
+import torch
+from torch.fx import GraphModule
+import torch._dynamo as torchdynamo
+from torch.nn.utils.fusion import fuse_conv_bn_weights
+
+from torch.ao.quantization.fx.prepare import (
+    is_activation_post_process_node,
+)
+from .qconfig_mapping import QConfigMapping
+from .backend_config import BackendConfig
+from .fx import prepare
+from .quantize_fx import _convert_to_reference_decomposed_fx
+
+from typing import Tuple, Any, Dict
+from collections import OrderedDict
+import operator
+
+# TODO: longer term, don't need to retrace or parse the string
+# we should have node.meta["nn_module_stack"] that store the dict
+def _infer_nn_stack_trace_and_append_on_mode(m, gm, args_as_list):
```

Comment:
nit: can we move the utils to a utils file, and keep this file to user facing functions only?

Reply:
sure